lstchain/scripts/lstchain_mc_rfperformance.py

```diff
@@ -111,6 +113,10 @@ def main():
 
     selected_gammas = dl2.query('reco_type==0 & mc_type==0')
 
+    if(len(selected_gammas) == 0):
+        log.warning('No gammas selected, I will not plot any output') 
+        sys.exit()
```

Comment:
default sys.exit() is a 0 exit code - meaning that the code will exit without prompting any error. If you want that the code exits with an error, you should add `sys.exit(1)`.
However, I'm not sure whether a unitary test will fail or not if the exit error is 0.

Reply:
yeah, that was exactly the problem, if the script exits with an error, then the tests do not pass. Although this is a very particular case, I precisely found it while trying to solve the first problem. 
Maybe I could raise here a warning, but what is really needed is that the code does not continue executing. If you have a suggestion, it would be great

Reply:
Ok, nevermind. Of course if the code exits with and error, the test will not pas... 😅

Reply:
(sorry for the noise, I'm mixing things between the exit kind of the script and the unit test)

Reply:
no problem, reviews are always very welcome! ;-)